src/LongTextField.tsx

```diff
@@ -42,6 +43,8 @@ function LongTextField({
   onChange,
   placeholder,
   value,
+  rows,
+  autoResize,
```

Comment:
It would be convenient to set defaults for these properties, so the existing behavior is not altered.

Reply:
As discussed, these parameters allow for `undefined` values, so we're keeping them as stated in the PR